src/Form/LegacyConsumer/templates/base.html.php

```diff
@@ -1,9 +1,9 @@
-<?php /** @var Give\Framework\FieldsAPI\Field $field */ ?>
+<?php /** @var Give\Framework\FieldsAPI\Text $field */ ?>
```

Comment:
Please revert this change, this isn’t always a `Text` type. It needs to be `Field`. If you want to make it work for your IDE, then you can list out each trait that is used separated with `|`. I did that before, but I found the comment was super long with the fully qualified names.

Reply:
fixed: https://github.com/impress-org/givewp/commit/06b5fc671754cf93b4e2c172ee1587ba993c8d61